test/adapter/json_api/has_many_test.rb

```diff
@@ -48,11 +48,23 @@ def test_includes_empty_comments_linkage
 
           def test_allow_blank_linkage_option_set_to_false
             @post.comments = []
-            @adapter = ActiveModel::Serializer::Adapter::JsonApi.new(@serializer, include: 'bio,posts', include_blank_linkage: false)
+            @adapter = ActiveModel::Serializer::Adapter::JsonApi.new(@serializer, include_blank_linkage: false)
```

Comment:
Because include does not make much sense here. `include` is used for the `included` objects, not linkage and I'm only testing linkage here..